source/guides/token-management/index.markdown

```diff
@@ -197,6 +204,14 @@ $result = $auth->authenticate($passwordGrant);
 
 //Return Type: Stormpath\Oauth\OauthGrantAuthenticationResult
 ------
+PasswordGrantRequest createRequest = Oauth2Requests.PASSWORD_GRANT_REQUEST.builder()
+  .setLogin("username@test.com")
+  .setPassword("Password1!")
+  .build();
+OauthGrantAuthenticationResult oauthGrantAuthenticationResult = Authenticators.PASSWORD_GRANT_AUTHENTICATOR
+  .forApplication(app)
+  .authenticate(createRequest);
+------
 {% endcodetab %}
 
 {% docs note %}
```

Comment:
Please see below here: "against an particular" should be "against a particular".